cmd/ack-generate/command/release.go

```diff
@@ -74,21 +73,7 @@ func generateRelease(cmd *cobra.Command, args []string) error {
 	if err := ensureSDKRepo(ctx, optCacheDir, optRefreshCache); err != nil {
 		return err
 	}
-	sdkHelper := ackmodel.NewSDKHelper(sdkDir)
-	sdkAPI, err := sdkHelper.API(svcAlias)
-	if err != nil {
-		newSvcAlias, err := FallBackFindServiceID(sdkDir, svcAlias)
-		if err != nil {
-			return err
-		}
-		sdkAPI, err = sdkHelper.API(newSvcAlias) // retry with serviceID
-		if err != nil {
-			return fmt.Errorf("service %s not found", svcAlias)
-		}
-	}
-	m, err := ackmodel.New(
-		sdkAPI, "", optGeneratorConfigPath, ackgenerate.DefaultConfig,
-	)
+	m, err := loadModel(svcAlias, "")
```

Comment:
nit: could you add a comment as to why the apiVersion is empty here? If it doesn't matter, then `loadModelWithLatestAPIVersion` would be clearer

Reply:
@A-Hilaly Would love your input on this.